templates/branding/openSUSE/external_reporting.html.ep

```diff
@@ -35,9 +38,13 @@
 %         $product = join(' ', $flavor_to_prod_sle{$subproduct}, $version);
 %     }
 % }
-% else {
+% elsif ($job->DISTRI eq 'opensuse') {
 %     $product = $job->VERSION eq 'Tumbleweed' ? 'Tumbleweed' : 'Distribution';
 % }
+% elsif ($job->DISTRI eq 'casp') {
+%     # sic!
+%     $product = "1 .0\x{C2A0}(CASP/MicroOS)";
```

Comment:
the `\x{C2A0}` for non-breaking space does not really work, gives me a chinese hut instead. any ideas?